### PR TITLE
use variable open sans font instead of several static google fonts.

### PR DESCRIPTION
**Pull request by staabm**
untested change!

closes https://github.com/mnapoli/externals/issues/80

- reduces number of required resources to load
- reduces number of external services (less dns etc)

### CODE REVIEW

File: web/fonts/SourceSansVariable-Roman_LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Copyright 2010, 2012, 2014 Adobe Systems Incorporated (http://www.adobe.com/), with Reserved Font Name 'Source'. All Rights Reserved. Source is a trademark of Adobe Systems Incorporated in the United States and/or other countries.
```
Comment (staabm) [inline web/fonts/SourceSansVariable-Roman_LICENSE.txt]: not sure it makes sense we have the license in the repos, but I thought it might make sense

**Comment by staabm** [comment]
https://github.com/mnapoli/externals/pull/92 should be merged/discarded before this PR.

**Comment by staabm** [comment]
Conflicts resolved

**Comment by mnapoli** [comment]
I gave it a try locally but the font-weight doesn't seem to be taken into account, so I'm not sure if the font loads correctly. I don't know if it's that but the font doesn't look a lot like Open Sans to be honest (but again, I'm not sure if it's working).

**Comment by staabm** [comment]
I guess we should wait with this PR until plattform.sh fixed their api issues and I can test this properly

**Comment by staabm** [comment]
It seems the variable font would only be worth it, when a lot of different page weights are used.

Atm we use only a few. Also I would love to get rid of some of them so we can use less regular font files

**Pull request closed without merging**